src/dns_conf.c

```diff
@@ -650,6 +653,15 @@ static int _conf_domain_rule_ipset(char *domain, const char *ipsetname)
 			continue;
 		}
 
+		// replace '/' to ' ' so as to support nftset
+		char* i = tok;
```

Comment:
This code should not be used. This processing may cause the ipset function to fail. When adding ipset later, you can get the IP and ipset name.